### PR TITLE
Use React context instead of context prop

### PR DESCRIPTION
**Pull request by mridgway**
Required update to fluxible@0.2.0

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by jedireza** [comment]
:+1: that looks much better now

File: components/Docs.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ var Component = React.createClass({
                 <div className="content">
                     <div className="pure-g">
                         <div className="pure-u-1 pure-u-md-5-24">
-                            <Menu config={this.props.menu} selected={this.props.doc.key} context={context} />
+                            <Menu config={this.props.menu} selected={this.props.doc.key} />
                         </div>
                         <div className="pure-u-1 pure-u-md-19-24">
                             <Doc content={this.props.doc.content} context={context} />
```
Comment (lingyan) [inline components/Docs.jsx]: remove this `context = {context}` as well

**Comment by lingyan** [comment]
Left one comment.  Otherwise looks good.

**Comment by redonkulus** [comment]
:shipit:

**Comment by redonkulus** [comment]
Please update docs.

**Comment by redonkulus** [comment]
:shipit:

File: server.js [hunk 2]
```diff
@@ -62,16 +62,16 @@ server.use(function (req, res, next) {
...
-            })),
-            tracking: tracking
-        }));
-
-        res.send(doctype + html);
+        React.withContext(context.getComponentContext(), function () {
+            var html = React.renderToStaticMarkup(HtmlComponent({
+                state: exposed,
+                markup: React.renderToString(AppComponent({
+                    context: context.getComponentContext()
```
Comment (kennydee) [inline server.js]: Why sending again the context here ? Thanks

Comment (redonkulus) [inline server.js]: @kennydee That is actually not needed, it should be removed.

Comment (kennydee) [inline server.js]: :+1: Thanks

Comment (redonkulus) [inline server.js]: @kennydee removed https://github.com/yahoo/fluxible.io/commit/348780b8ca9f342548db77753b6d3131d50c3510

**Pull request merged**